pattern-lab/source/_patterns/_patterns-global/scss/admin/layout_builder_modal.scss

```diff
@@ -322,6 +322,31 @@ ul.paragraphs-add-dialog-list input.field-add-more-submit {
         border: 2px solid $color-primary-hover!important;
       }
     }
+
+    .form-actions {
+      background: transparent;
+
+      .button.button--primary  {
+        display: block!important;
```

Comment:
Please explain why we are using !important here.

Reply:
By-default on the inline style is applied on the button to override this need to apply !important here

Reply:
Please check why an inline style of `display:none` is getting attached on Save button for a media entity. This should not be the case.

Please check the root cause of the issue and fix accordingly.